modules/nextflow/src/main/groovy/nextflow/cli/CmdTrace.groovy

```diff
@@ -202,18 +202,18 @@ class CmdCid extends CmdBase implements UsageAware {
         @Override
         void usage() {
             println description
-            println "Usage: nextflow $NAME $name <CID reference> "
+            println "Usage: nextflow $NAME $name <Lineage ID> "
         }
     }
 
-    class CmdLineage implements SubCmd {
+    class CmdTrace implements SubCmd {
 
         @Override
-        String getName() { 'lineage' }
+        String getName() { 'trace' }
```

Comment:
Maybe "render"? also because the description says that

Reply:
Agree

Reply:
I think `render` is too similar to `show`, better to be explicit that you are "tracing" the lineage rather than showing it

Reply:
IMO it's preferable "render" because is simpler, more correlated with the idea of rendering the lineage, and also because "trace/tracing" is already used for logs, for nextflow [trace](https://www.nextflow.io/docs/latest/reference/config.html#trace) file, etc

Reply:
It's too similar to `show`, it will confuse people

Reply:
Let's take this risk

Reply:
I'm ok with this, @jorgee what do you agree?

Reply:
Yes, fine with me. So, trace->render and show->describe.

Reply:
👍